test/UnitTests/Microsoft.Testing.Platform.UnitTests/Program.cs

```diff
@@ -19,8 +19,10 @@
 #if ENABLE_CODECOVERAGE
 builder.AddCodeCoverageProvider();
 #endif
-builder.AddCrashDumpProvider();
+builder.AddAppInsightsTelemetryProvider();
+builder.AddCrashDumpProvider(ignoreIfNotSupported: true);
```

Comment:
I'm curious why ignoreIfNotSupported was not needed previously

Reply:
Probably because crash dump is passive (unlike hang dump), so what happens is that on activation you set env variables and hope that some crash dumps will appear in the dump folder (those will be created by the built-in .net crash reporter). This setup won't fail in native aot, and we probably don't test that it actually works there because we don't expect it to work?

If the process was more active, like in VSTest where we use procdump for .net framework processes, then even the setup would probably fail in native aot, or other incompatible scenario, and we would see it fail early, even without explicitly testing for it.